plugins/Plugin.js

```diff
@@ -0,0 +1,162 @@
+class GiftPlanet {
```

Comment:
Let's remove this, I think the default plugin template is good enough

Reply:
Sure, removed.